pedometer/chapter.md

```diff
@@ -11,77 +13,79 @@ Let's roll up our sleeves, and prepare to untangle.
 
 Our total acceleration is the sum of user acceleration and gravitational acceleration.
 
-![](chapter-figures/component-signals-2.png)\
+![](chapter-figures/component-signals-2.png)
 
-To count steps, we're interested in the bounces created by the user in the direction of gravity. That means we're interested in isolating the 1-dimensional time series **user acceleration in the direction of gravity** from our 3-dimensional acceleration signal.
+To count steps, we're interested in the bounces created by the user in the direction of gravity. That means we're interested in isolating the 1-dimensional time series "user acceleration in the direction of gravity" from our 3-dimensional acceleration signal.
 
-When our stick person is walking with step bounces in the y direction, gravitational acceleration is zero in $x(t)$ and $z(t)$ and constant at $9.8 m/s^2$ in $y(t)$, so $x(t)$ and $z(t)$ fluctuate around 0 while $y(t)$ fluctuates around $-1g$. Note the obvious peaks in $y(t)$. Those are due to step bounces! Looking at the user acceleration plots next, we notice that when we remove gravitational acceleration from the time series, all three time series fluctuate around 0. In our last plot, gravitational acceleration, $y_{g}(t)$ is constant at $-1g$, $x_{g}(t)$ and $z_{g}(t)$ are constant at 0.
+When our stick person is walking with step bounces in the $y$ direction, gravitational acceleration is 0 in $x(t)$ and $z(t)$ and constant at 9.8 $m/s^2$ in $y(t)$, so $x(t)$ and $z(t)$ fluctuate around 0 while $y(t)$ fluctuates around -1 *g*. Note the obvious peaks in $y(t)$. Those are due to step bounces! 
+
+Looking at the user acceleration plots next, we notice that --- because we have removed gravitational acceleration --- all three time series fluctuate around 0. In our last plot, gravitational acceleration, $y_{g}(t)$ is constant at -1 *g*, and $x_{g}(t)$ and $z_{g}(t)$ are constant at 0.
 
 So, in our simple case, the 1-dimensional user acceleration in the direction of gravity time series we're interested in is $y_{u}(t)$. Although $y_{u}(t)$ isn't as smooth as our perfect sine wave, we can identify the peaks, and use those peaks to count steps. So far, so good. Now, let's add a little more reality to our world.
 
 ## People Are Complicated Creatures
 
-Let's see what happens when we introduce real people into the mix. What if a person carries the phone in a bag on their shoulder, with the phone in a more wonky position? To make matters worse, what if the phone rotates in the bag part way through the walk?
+Let's see what happens when we introduce more complications. What if a person carries the phone in a bag on their shoulder, with the phone in a more wonky position? To make matters worse, what if the phone rotates in the bag part way through the walk?
```

Comment:
Changed this because the shirt pocket person was at least as real as this shoulder bag person. :)